src/main/java/org/testng/junit/JUnit4TestRunner.java

```diff
@@ -222,6 +227,16 @@ private void  runAfterInvocationListeners(ITestResult tr) {
                 l.afterInvocation(im, tr);
             }
         }
+
+        private void validate(ITestResult tr, Description description) {
```

Comment:
can be static

Reply:
Inner class methods cannot be static.

Reply:
Ok, I didn't know

Reply:
I also didn't know that until IntelliJ complained :)